tests.sh

```diff
@@ -349,7 +340,7 @@ build_php5.5() {
 
 build_php5.5_c() {
   use_php 5.5
-  wget https://phar.phpunit.de/phpunit-4.8.0.phar -O /usr/bin/phpunit
+  ls /usr/local/php-5.5/bin
```

Comment:
Do we want to keep the `ls` call, or is this for debugging?

Reply:
Removed.